frontend/src/HomePage/ExportAppModal.jsx

```diff
@@ -164,14 +191,14 @@ export default function ExportAppModal({ title, show, closeModal, customClassNam
               className="import-export-footer-btns"
               variant="tertiary"
               data-cy="export-all-button"
-              onClick={() => exportApp(app, null, exportTjDb, tables)}
+              onClick={() => exportApp(app, null, exportTjDb, tables, isDownload = 'true')}
```

Comment:
Can't we simply pass `tables` or `allTables` here itself instead of a new parameter?

Reply:
@akshaysasidrn yes, this makes sense, will change that.